src/crimson/osd/pg.cc

```diff
@@ -907,15 +907,30 @@ void PG::mutate_object(
   }
 }
 
+void PG::enqueue_push_for_backfill(
+  const hobject_t &obj,
+  const eversion_t &v,
+  const std::vector<pg_shard_t> &peers)
+{
+  assert(recovery_handler);
+  assert(recovery_handler->backfill_state);
+  auto backfill_state = recovery_handler->backfill_state.get();
+  backfill_state->enqueue_standalone_push(obj, v, peers);
+}
+
 PG::interruptible_future<
   std::tuple<PG::interruptible_future<>,
              PG::interruptible_future<>>>
 PG::submit_transaction(
   ObjectContextRef&& obc,
+  ObjectContextRef&& new_clone,
   ceph::os::Transaction&& txn,
-  osd_op_params_t&& osd_op_p,
+  osd_op_params_t&& opp,
   std::vector<pg_log_entry_t>&& log_entries)
 {
+  auto _new_clone = std::move(new_clone);
+  auto osd_op_p = std::move(opp);
+  auto at_version = osd_op_p.at_version;
```

Comment:
Can you rebase on top of https://github.com/ceph/ceph/pull/60596 here? As the fix here is not related to the commit.  I'll try to merge 60596 soon.

Reply:
Yeah, will do.